sdcm/results_analyze.py

```diff
@@ -872,7 +872,9 @@ def check_regression_with_subtest_baseline(self, test_id, base_test_id, subtest_
                        )
         self.log.debug('Regression analysis:')
         self.log.debug(PP.pformat(results))
-        test_name = full_test_name.split('.', 1)[1]  # Example: longevity_test.py:LongevityTest.test_custom_time
+
+        test_name = full_test_name.split('.', 1)  # Example: longevity_test.py:LongevityTest.test_custom_time
+        test_name = test_name[1] if len(test_name) > 1 else full_test_name
```

Comment:
please add examples here if we really need that

Reply:
removed.